src/components/tiles/LtvRatioTile.tsx

```diff
@@ -66,5 +66,10 @@ export const LtvRatioTile = (props: {
 }
 
 export const getLtvColorValue = (ltvValue: string, theme: Theme): string => {
-  return gt(ltvValue, '0.7') ? theme.dangerText : gt(ltvValue, '0.6') ? theme.warningText : theme.positiveText
+  try {
+    return gt(ltvValue, '0.7') ? theme.dangerText : gt(ltvValue, '0.6') ? theme.warningText : theme.positiveText
```

Comment:
Why is it possible `ltvValue` can be `NaN`? Seems like the right solution is to fix the data being passed into the function rather than handling invalid data types.

Reply:
Due to dividing by 0 with numbers in BorrowEngineFactory. Fix has been updated to fix the data while still retaining a separate display value/color for fresh loans where LTV cannot yet be calculated.